src/platform/plugins/shared/unified_doc_viewer/public/components/observability/attributes/doc_viewer_attributes_overview/group_attributes_fields.ts

```diff
@@ -43,17 +40,12 @@ export function groupAttributesFields({
     if (!lowerFieldName.includes(lowerSearchTerm)) return;
     if (isEsqlMode && areNullValuesHidden && flattened[fieldName] == null) return;
 
-    const field: AttributeField = {
-      name: fieldName,
-      displayName: getAttributeDisplayName(fieldName),
-    };
-
     if (lowerFieldName.startsWith('resource.attributes.')) {
-      resourceAttributesFields.push(field);
+      resourceAttributesFields.push(fieldName);
```

Comment:
nit: since we're no longer customizing the field name, I'm wondering if this function should be responsible for returning the entire DataTableRecord instead of just the field.